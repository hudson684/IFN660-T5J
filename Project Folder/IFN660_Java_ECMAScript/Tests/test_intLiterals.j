0x7fff_ffff
0177_7777_7777
0b0111_1111_1111_1111_1111_1111_1111_1111

0x8000_0000
0200_0000_0000
0b1000_0000_0000_0000_0000_0000_0000_0000

0xffff_ffff
0377_7777_7777
0b1111_1111_1111_1111_1111_1111_1111_1111

0x7fff_ffff_ffff_ffffL
07_7777_7777_7777_7777_7777L
0b0111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111L

0x8000_0000_0000_0000L
010_0000_0000_0000_0000_0000L
0b1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000L

0xffff_ffff_ffff_ffffL
017_7777_7777_7777_7777_7777L
0b1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111L

0
2
0372
0xDada_Cafe
1996
0x00_FF__00_FF

0l
0777L
0x100000000L
2_147_483_648L
0xC0B0L