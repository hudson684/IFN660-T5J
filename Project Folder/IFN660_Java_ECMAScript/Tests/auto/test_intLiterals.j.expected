IntegerLiteral (2147483647)
IDENT (test2)
IntegerLiteral (-2147483648)
IntegerLiteral (-2147483648)
IntegerLiteral (-2147483648)
IDENT (test3)
IntegerLiteral (4294967295)
IntegerLiteral (4294967295)
IntegerLiteral (4294967295)
IDENT (test4)
IntegerLiteral (9223372036854775807)
IntegerLiteral (9223372036854775807)
IntegerLiteral (9223372036854775807)
IDENT (test5)
IntegerLiteral (-9223372036854775808)
IntegerLiteral (-9223372036854775808)
IntegerLiteral (-9223372036854775808)
IDENT (test6)
IntegerLiteral (-1)
IntegerLiteral (-1)
IntegerLiteral (-1)
IDENT (test7)
IntegerLiteral (0)
IntegerLiteral (2)
IntegerLiteral (250)
IntegerLiteral (-623195394)
IntegerLiteral (1996)
IntegerLiteral (16711935)
IDENT (test8)
IntegerLiteral (0)
IntegerLiteral (511)
IntegerLiteral (4294967296)
IntegerLiteral (2147483648)
IntegerLiteral (49328)
EOF